package.json

```diff
@@ -24,6 +24,10 @@
   },
   "private": true,
   "dependencies": {
+    "@ag-grid-community/angular": "^29.2.0",
```

Comment:
These package references should not include the "^" symbol. By not not including, it will always lock `npm install` to those exact versions, allowing us to better control which versions get installed. It is also something that RenovateBot would attempt to fix anyway.